TypeError: codspeed_benchmark.<locals>.run() got an unexpected keyword argument 'c'

I was trying to run https://github.com/PyO3/pyo3/tree/main/pytests benchmarks locally with codspeed and run into this error.

With this patch

```diff
diff --git a/examples/word-count/requirements-dev.txt b/examples/word-count/requirements-dev.txt
index fcd8fc54c..7a597ac2a 100644
--- a/examples/word-count/requirements-dev.txt
+++ b/examples/word-count/requirements-dev.txt
@@ -1,2 +1,3 @@
 pytest>=3.5.0
 pytest-benchmark>=3.1.1
+pytest-codspeed>=1.2.0
diff --git a/pytests/requirements-dev.txt b/pytests/requirements-dev.txt
index d1fa05414..f971f2724 100644
--- a/pytests/requirements-dev.txt
+++ b/pytests/requirements-dev.txt
@@ -1,4 +1,5 @@
 hypothesis>=3.55
 pytest>=6.0
 pytest-benchmark>=3.4
+pytest-codspeed>=1.2.0
 psutil>=5.6
```

In `pytests` directory, run `nox -s bench -- --codspeed` to reproduce.

```
benchmark = <function codspeed_benchmark.<locals>.run at 0x10638e830>

    def test_simple_py(benchmark):
>       benchmark(simple_py, 1, "foo", c={1: 2})
E       TypeError: codspeed_benchmark.<locals>.run() got an unexpected keyword argument 'c'

tests/test_deprecated_pyfunctions.py:24: TypeError
```

--- a/src/pytest_codspeed/plugin.py
+++ b/src/pytest_codspeed/plugin.py
@@ -191,12 +191,14 @@ def pytest_sessionfinish(session: "pytest.Session", exitstatus):
 def codspeed_benchmark(request: "pytest.FixtureRequest") -> Callable:
     plugin = get_plugin(request.config)
 
-    def run(func: Callable[..., Any], *args: Any):
+    def run(func: Callable[..., Any], *args: Any, **kwargs: Any) -> Any:
         if plugin.is_codspeed_enabled and plugin.should_measure:
             assert plugin.lib is not None
-            _run_with_instrumentation(plugin.lib, request.node.nodeid, func, *args)
+            _run_with_instrumentation(
+                plugin.lib, request.node.nodeid, func, *args, **kwargs
+            )
         else:
-            func(*args)
+            func(*args, **kwargs)
 
     return run
 

--- a/tests/test_pytest_plugin.py
+++ b/tests/test_pytest_plugin.py
@@ -39,6 +39,19 @@ def test_plugin_enabled_without_env(pytester: pytest.Pytester) -> None:
     )
 
 
+def test_plugin_enabled_with_kwargs(pytester: pytest.Pytester, codspeed_env) -> None:
+    pytester.makepyfile(
+        """
+        def test_arg_kwarg_addition(benchmark):
+            def fn(arg, kwarg=None):
+                assert arg + kwarg == 40
+            benchmark(fn, 25, kwarg=15)
+        """
+    )
+    result = pytester.runpytest("--codspeed")
+    result.stdout.fnmatch_lines(["*1 benchmark tested*"])
+
+
 @skip_without_valgrind
 def test_plugin_enabled_by_env(pytester: pytest.Pytester, codspeed_env) -> None:
     pytester.makepyfile(

